armi/reactor/grids.py

```diff
@@ -923,9 +923,9 @@ def getLocatorFromRingAndPos(self, ring, pos, k=0):
         Parameters
         ----------
         ring : int
-            Ring index
+            Ring index (or number starting at 1 for hex)
```

Comment:
Ring and position are __always__ 1-based

Reply:
...and if this is inconsistent in any geometry, then they are wrong

Reply:
rzTheta seems to be wrong then:
```
    def getRingPos(self, indices):
        return (indices[1] + 1, indices[0] + 1)

    @staticmethod
    def getIndicesFromRingAndPos(ring, pos):
        return (pos, ring)
```

We are coding in python, so non-zero based indexing is surprising. Also the argument k is clearly zero based indexing, so I thought it would be helpful to point out that this is 1 based.

It is unclear from your feedback what actions you would like me to take. Can you give me some actionable feedback, or let me know if you are okay with these changes?

Reply:
Not really sure what you wanted, so I guessed. Let me know if this satisfies your concerns.

Reply:
rzTheta is wrong. 100% agree that ring and position being 1-based is surprising, but it __is__ 1-based. The greater sin is to be inconsistent within ring/pos. Could you fix rzTheta?

Reply:
I thought I did.